Assets/Scripts/UI/CraftingHoverTooltipResource.cs

```diff
@@ -0,0 +1,19 @@
+﻿using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using Utils;
+
+namespace UI
+{
+    public class CraftingHoverTooltipResource : MonoBehaviour
+    {
+        [SerializeField] private Image imgResource;
+        [SerializeField] private TextMeshProUGUI txtResource;
+
+        public void Init(ItemResourceData itemResourceData)
+        {
+            imgResource.sprite = itemResourceData.item.Icon;
+            txtResource.SetText($"{itemResourceData.amount} x {itemResourceData.item.ItemName}");
```

Comment:
Could you use the same x character Tobi used previously for the crafting recipes? I think it was this one, but I'm not totally sure: ⨉